packages/components/organisms/f-header/package.json

```diff
@@ -40,12 +40,13 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.3.0",
-    "@justeat/f-vue-icons": "1.4.0"
+    "@justeat/f-services": "1.3.0"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "@justeat/f-trak": ">=0.6.0"
+    "@justeat/f-trak": ">=0.6.0",
+    "@justeat/f-vue-icons": "2.3.0",
+    "@justeat/f-button": "0.4.1"
```

Comment:
Same for the button I would think here too –  not sure you need to include it here, as it'll get built into the module at compile time.